ui/src/components/sermons/NoOnlineStreamPage.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import {
+  Container,
+  VStack,
+  Heading,
+  Text,
+  Button,
+  Box,
+  Icon,
+  useColorModeValue,
+} from '@chakra-ui/react';
+import { useHistory } from 'react-router-dom';
+import { MdVideoLibrary, MdSchedule } from 'react-icons/md';
+
+const NoOnlineStreamPage = () => {
+  const history = useHistory();
+  const bgColor = useColorModeValue('gray.50', 'gray.900');
+  const cardBg = useColorModeValue('white', 'gray.800');
+
+  const handleGoToSermons = () => {
+    history.push('/sermons');
+  };
+
+  return (
+    <Box bg={bgColor} minH="70vh" py={8}>
+      <Container maxW="container.md">
+        <VStack spacing={8} textAlign="center">
+          {/* Icon */}
+          <Box>
+            <Icon as={MdSchedule} w={16} h={16} color="blue.500" mb={4} />
+          </Box>
+
+          {/* Main Content */}
+          <Box bg={cardBg} p={8} borderRadius="lg" shadow="md" w="100%">
+            <VStack spacing={6}>
+              <Heading
+                as="h1"
+                size="xl"
+                fontFamily="DMSerifDisplay_Italic"
+                color="blue.600"
+              >
+                No Live Stream Available
+              </Heading>
+
+              <Text fontSize="lg" color="gray.600" maxW="md" lineHeight="tall">
```

Comment:
change the order of text below? i think this makes more sense:

No Live Stream Available
Our live streams typically happen....

You can access our past sermons...
Button to browse past sermons